scripts/load_templates.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
We need to add a call to this in all our scripts and also in the readme as a setup step if you're running anything manually